uaclient/cli.py

```diff
@@ -41,6 +44,21 @@
 
 STATUS_FORMATS = ["tabular", "json", "yaml"]
 
+UA_LOGS = (
+    "/var/log/ubuntu-advantage.log",
+    "/var/log/ubuntu-advantage-timer.log",
+    "/var/lib/ubuntu-advantage/jobs-status.json",
+    "/etc/apt/sources.list.d/ubuntu*",
+    "/var/lib/ubuntu-advantage/private/machine-id",
+)
+
+UA_SERVICES = (
+    "ua-timer.service",
```

Comment:
Remember if you land this after #1799 you'll also need the following:

ua-license-check.path
ua-license-check.service
ua-license-check.timer

Reply:
Can include now